packages/admin/admin/src/alert/Alerts.tsx

```diff
@@ -0,0 +1,176 @@
+import { ArrowRight, Check, Close, Error, Info, Warning } from "@comet/admin-icons";
+import { Button, ButtonProps, IconButton, Theme, Typography } from "@mui/material";
+import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import clsx from "clsx";
+import * as React from "react";
+
+type Action = Partial<Omit<ButtonProps, "children">> & {
+    text: ButtonProps["children"];
+};
+
+export interface AlertProps {
+    severity?: "info" | "warning" | "error" | "success";
+    title?: React.ReactNode;
+    text: React.ReactNode;
+    hideCloseButton?: boolean;
+    onCloseClick?: () => void;
+    action?: Action;
+}
+
+export type AlertClassKey =
+    | "root"
+    | "severityInfo"
+    | "severityWarning"
+    | "severityError"
+    | "severitySuccess"
+    | "title"
+    | "text"
+    | "button"
+    | "container"
+    | "closeIcon"
+    | "hasTitle";
+
+const styles = (theme: Theme) =>
+    createStyles<AlertClassKey, AlertProps>({
+        root: {
+            display: "flex",
+            alignItems: "center",
+            backgroundColor: theme.palette.background.paper,
+            borderRadius: 4,
+            width: 400,
+            padding: 20,
+            boxShadow: "0px 0px 20px 0px rgba(0, 0, 0, 0.10)",
+            position: "relative",
+        },
+        title: {
+            fontWeight: "bold",
```

Comment:
This should be `600` according to the Design, `bold` is `700` 🙂 

```suggestion
            fontWeight: 600,
```